cpg-core/src/main/kotlin/de/fraunhofer/aisec/cpg/graph/Extensions.kt

```diff
@@ -612,6 +612,27 @@ val Node?.assigns: List<AssignExpression>
 val Node?.problems: List<ProblemNode>
     get() = this.allChildren()
 
+/**
+ * Returns all [Node] children in this graph, where the [Node.additionalProblems] field is not
+ * empty.
+ */
+val Node?.additionalProblemsNotEmpty: List<Node>
```

Comment:
nodesWithProblems?

Reply:
Please check the naming again. I'm quite happy with the current name for the additional problem nodes. However, the name for the problem nodes directly in the graph is a bit misleading. Suggestions?